Doc/data/refcounts.dat

```diff
@@ -1284,6 +1284,19 @@ PyLong_FromUnsignedLong:unsignedlong:v::
 PyLong_FromVoidPtr:PyObject*::+1:
 PyLong_FromVoidPtr:void*:p::
 
+PyLong_IsPositive:int:::
+PyLong_IsPositive:PyObject*:obj:0:
+
+PyLong_IsNegative:int:::
+PyLong_IsNegative:PyObject*:obj:0:
+
+PyLong_IsZero:int:::
+PyLong_IsZero:PyObject*:obj:0:
+
+PyLong_GetSign:int:::
+PyLong_GetSign:PyObject*:v:0:
+PyLong_GetSign:int*:sign::
```

Comment:
Lets remove this for now.  I'm not sure about fate of this function.

Reply:
It seems to have been merged, and I am a little hesitant to delete it. But if I remember correctly, it was discussed to remove it, so we can wait for victor's speech. WDYT?

Reply:
If PyLong_GetSign() is removed, it can be removed from Doc/data/refcounts.dat as well.